CRM/Contact/BAO/Relationship.php

```diff
@@ -1288,10 +1291,15 @@ public static function getRelationship(
       while ($relationship->fetch()) {
         $relationshipCount += $relationship->cnt1 + $relationship->cnt2;
       }
+      CRM_Core_DAO::executeQuery("DROP TABLE IF EXISTS civicrm_contact_relationships");
```

Comment:
You need to make it DROP TEMPORARY TABLE - could be causing the test fails as it will block rollback without the keyword